src/reanimated2/getShadowNodeWrapperFromRef.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable camelcase */
+
+import { ShadowNodeWrapper } from './hook/commonTypes';
+
+// Used by GestureDetector (unsupported on web at the moment) to check whether the
+// attached view may get flattened on Fabric. This implementation causes errors
+// on web due to the static resolution of `require` statements by webpack breaking
+// the conditional importing. Solved by making .web file.
```

Comment:
I think this comment may be removed (or at least updated as there's no `.web` file, and `GestureDetector` is in the other library).

Reply:
Removed in 2d43f6255e928f8e4dd6203754819da3de297315